projects/configs/test-extract-uclh-omop-cdm-xnat.yaml

```diff
@@ -0,0 +1,37 @@
+#  Copyright (c) 2024 University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+project:
+    name: "test-extract-uclh-omop-cdm-xnat"
+    azure_kv_alias: "test"
+    modalities: ["DX", "CR"]
```

Comment:
I know its a copy of what already exists but strange that this is doesn't have `MR` in there

Reply:
ah well spotted, I've added it in